README.md

```diff
@@ -188,6 +189,28 @@ Short `TODO` List for new contributors:
 - Increment the test coverage.
 - [Issues marked as `Help wanted`](https://github.com/ChuckerTeam/chucker/labels/help%20wanted)
 
+### Building 🛠
```

Comment:
Probably, we should update this section to mention about pre-commit hook from #408

Reply:
Updated